xtras/src/send_interval.rs

```diff
@@ -14,27 +14,42 @@ where
     /// handler returns before scheduling a new one, thus preventing them from
     /// piling up.
     /// As a bonus, this function is non-fallible.
-    async fn send_interval<F>(self, duration: Duration, constructor: F)
+    async fn send_interval<F>(self, duration: Duration, constructor: F, verbosity: TraceVerbosity)
     where
         F: Send + Sync + Fn() -> M,
         A: xtra::Handler<M, Return = ()>;
 }
 
+/// How verbose a given trace will be. If it is set to quiet, it will be disabled, alongside all
+/// of its children.
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+pub enum TraceVerbosity {
+    Quiet,
+    Normal,
+}
+
 #[async_trait]
 impl<A, M> SendInterval<A, M> for address::Address<A>
 where
     A: xtra::Handler<M>,
     M: Send + 'static,
 {
-    async fn send_interval<F>(self, duration: Duration, constructor: F)
+    async fn send_interval<F>(self, duration: Duration, constructor: F, verbosity: TraceVerbosity)
     where
         F: Send + Sync + Fn() -> M,
     {
         let span = || {
-            tracing::debug_span!(
-                "Send message every interval",
-                interval_secs = %duration.as_secs()
-            )
+            if verbosity == TraceVerbosity::Quiet {
+                tracing::debug_span!(
+                    "Send message every interval (quiet)",
```

Comment:
I don't really understand this span description. What does sending quietly mean? I thought it just won't be sent.

Reply:
This means that it and all of its children won't be exported over OTEL